src/DataProtection/Cryptography.Internal/src/Cng/OSVersionUtil.cs

```diff
@@ -11,34 +12,29 @@ internal static class OSVersionUtil
 
     private static OSVersion GetOSVersion()
     {
-        const string BCRYPT_LIB = "bcrypt.dll";
-        SafeLibraryHandle? bcryptLibHandle = null;
-        try
-        {
-            bcryptLibHandle = SafeLibraryHandle.Open(BCRYPT_LIB);
-        }
-        catch
+        if (Environment.OSVersion.Platform is not PlatformID.Win32NT)
```

Comment:
This change looks fine but we have a much better API for OS version checking (https://docs.microsoft.com/en-us/dotnet/api/system.operatingsystem?view=net-6.0)